plutus-core-spec/plutus-core-specification.tex

```diff
@@ -970,84 +971,85 @@ \section{Built in types, functions, and values}
 %
 
   \item We use fixed Scott encodings for certain types and values,
```

Comment:
I don't think we actually have any occurrences of unit any more, but we do still have bool so I guess it doesn't hurt to leave it in (seems like it might pop up in future).

Reply:
I also think it's worth mentioning that these types are *not* used abstractly, so e.g. we will directly apply boolean values to do pattern matching.